webportal/lambda/.chalice/deployed/dev.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
These were originally .gitignored (but previous versions existed in github). It was dangerous to .gitignore these, as this is how chalice knows what is out there in and deployed.